docs/adrs/0004-bulk-import-output-cells.md

```diff
@@ -0,0 +1,51 @@
+# Bulk Import Output Cells
+
+Date: 2021-10-05
+
+One of the features of the App Cell is that it can produce Output Cells after a run. These typically contain views of the data that was transformed via the app, including new calculations being made or new data being created. However, in the Bulk Import cell, this could result in dozens or more new cells all being created at once, which could overwhelm a user or lead to other confusion from what should be an easy upload process.
+
+This ADR details the decision to not provide output cells for those apps run by the Bulk Import cell.
+
+## Author
+
+@briehl
+
+## Status
+
+Accepted
+
+## Alternatives Considered
+
+* Include Output Cells when Apps finish
+* Don't include Output Cells when Apps finish
+* Group Output Cells in a single, tabbed Output
+
+## Decision Outcome and Consequences
+
+The Bulk Import cell will not produce new Output Cells on app completion. It will, instead, ensure that there is a Report view available for each imported object. This carries the consequence that users may expect an output cell for one or two uploaded object types based on the single uploader. Extra documentation should help with that.
+
+A workaround could be to guide the user to choose one or more generated data objects to view by clicking on them in the data panel on the left side of the Narrative.
```

Comment:
Current importer apps that employ an output cell:
- Import GenBank file - genome viewer
- Import GFF/FASTA - genome viewer

Clicking on the object name in the "Objects" section of the result cell will also create a genome viewer.

OT: mysteriously, some of the single/paired end reads importers do create a reads viewer and others don't. ???